src/lib/components/button.svelte

```diff
@@ -18,18 +28,42 @@
 
 <style lang="scss">
 	button {
-		background: var(--color-background);
+		background: var(--color-content-bg);
 		margin: 0;
 		font-size: var(--font-size-button);
 		font-weight: var(--font-weight-button);
 		cursor: pointer;
 		display: flex;
-		flex-direction: row;
 		justify-content: center;
+		flex-direction: row;
 		align-items: center;
 		gap: var(--spacing-12);
 		padding: 10px;
 		border: none;
+		overflow-wrap: normal;
+
+		&.center {
+			margin-inline: auto;
+		}
+
+		&.left {
+			margin-left: 0;
+			margin-right: auto;
+		}
+
+		&.right {
+			margin-left: auto;
+			margin-right: 0;
+		}
+
+		&.reverse {
+			flex-direction: row-reverse;
+		}
```

Comment:
Is this because of having icon on the right side of button? I would consider instead to have two icon properties instead `iconStart` and `iconEnd`. This is more consistent with UI frameworks.

Reply:
Here is an example from Material UI (React) https://mui.com/material-ui/react-button/#buttons-with-icons-and-label

Or we can (and this may be better) change it from `props` to `slot`.

Reply:
I haven't seen a design for this page so I'm not sure what it's supposed to look like. What are you using as reference? Can you send me a link?

Reply:
I assumed you add this so that you can have the icon after the label, right? If my assumption is wrong then please explain why we have this :)

Reply:
Your assumption is correct 😁

Reply:
OK then I would prefer to either use slots for the icons or props.

```html
<Button>
   <Icon slot='start' />
    label
   <Icon slot='end' />
</Button>

<!-- OR -->
<Button iconStart={Icon} iconEnd={Icon}>
    label
</Button>
```

I have slight preference to use the slot variant

Reply:
Yup we can do that, let's chat so I understand the reason for the preference so I can make more informed decisions moving forward 😁